zookeeper-api/src/main/java/org/apache/helix/zookeeper/zkclient/ZkClient.java

```diff
@@ -1073,10 +1073,24 @@ protected void processStateChanged(WatchedEvent event) {
 
     fireStateChangedEvent(event.getState());
 
+    // This following is the case for sharedZkClient
     if (!isManagingZkConnection()) {
+      /*
```

Comment:
Comment block format:
```
/*
 * comments
 */
```

Reply:
I think we have the format style file? You can apply the style file in intellij.

Reply:
I applied the style file. And it did not pick this up.